src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        int number = new Random().nextInt(100);
```

Comment:
Lets make ColorSupplier and Random class level variables to avoid creating a new instance each time this method is called

Avoid magic numbers - add a constant for 100